tests/stress/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:10.12.0
```

Comment:
Change to 10.13.0

Reply:
cool, thanks @geoffturk !!!!

Reply:
This was already done on master yesterday.